package.json

```diff
@@ -2,20 +2,35 @@
   "name": "jquery-carousel",
   "version": "0.0.1",
   "description": "simple jquery-carousel",
+  "main": "src/js/cc_script.js",
```

Comment:
@Mariam13 please change main file path relative to `dist` directory, where our built artifacts are